hopr/hopr-lib/src/lib.rs

```diff
@@ -828,18 +828,23 @@ impl Hopr {
 
             // TODO: allow announcing all addresses once that option is supported
             let multiaddresses_to_announce = self.transport_api.announceable_multiaddresses();
-            info!("Announcing node on chain: {:?}", &multiaddresses_to_announce[0]);
-            if self
+
+            // The announcement is intentionally not awaited until confirmation
+            match self
                 .chain_api
                 .actions_ref()
                 .announce(&multiaddresses_to_announce[0], &self.me)
                 .await
-                .is_err()
             {
+                Ok(_) => info!("Announcing node on chain: {:?}", &multiaddresses_to_announce[0]),
+                Err(CoreEthereumActionsError::AlreadyAnnounced) => warn!(
```

Comment:
Is this really a warning? It should be info!, right? Because e.g. a restart turned the node off.